Products/ZenUI3/browser/resources/js/zenoss/process.js

```diff
@@ -101,7 +94,12 @@ var ProcessTreePanel = Ext.extend(Zenoss.HierarchyTreePanel, {
             cls: 'x-tree-noicon',
             directFn: router.getTree,
             router: router,
-            selModel: selModel,
```

Comment:
all selection models should be defined in owner component scopes to initialize it's functionality right.